src/components/molecule_new_word_box/index.tsx

```diff
@@ -22,6 +22,11 @@ const NewWordBox: FC = () => {
   useKeyPress(`Escape`, handleClickAddWord)
   const ref = useOutsideClicked(handleClickAddWord)
 
+  const handleClickCard = useCallback(
+    () => setWritingMode(true),
+    [setWritingMode],
+  )
```

Comment:
Make usePostWordWithStringHook() to return openWritingModeTrue
so that you do not have to make this callback function here